bundles/persistence/org.openhab.persistence.dynamodb/.classpath

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/aws-java-sdk-core-1.11.213.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/aws-java-sdk-dynamodb-1.11.213.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/aws-java-sdk-kms-1.11.213.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/aws-java-sdk-s3-1.11.213.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/commons-codec-1.9.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/aws-java-sdk-core-1.11.457.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/aws-java-sdk-dynamodb-1.11.457.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/aws-java-sdk-kms-1.11.457.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/aws-java-sdk-s3-1.11.457.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/commons-codec-1.10.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/commons-logging-1.1.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/httpclient-4.5.2.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/httpcore-4.4.4.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpclient-4.5.5.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpcore-4.4.9.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/ion-java-1.0.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jackson-annotations-2.6.0.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jackson-core-2.6.7.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/jackson-databind-2.6.7.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/jackson-databind-2.6.7.jar"/>
```

Comment:
Everywhere else we have jackson-databind-2.6.7.2.jar So this is probably an mistake?

Furthermore, I did get `jackson-databind-2.6.7.2.jar` when following the README, copy-paste accident?